Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM elixir:1.12-alpine as builder
+
+ENV MIX_ENV prod
+ENV MAGIC_COOKIE=$magic_cookie
+
+RUN apk update && \
+    apk add -f curl git build-base nodejs yarn rust cargo
+
+COPY . /logflare
+
+WORKDIR /logflare/native/sqlparser_ex
+# Compile Rust
+RUN cargo build
+
+WORKDIR /logflare
+RUN mix do local.rebar --force, local.hex --force
+RUN mix do deps.get, deps.compile
+
+RUN mix phx.digest
+RUN mix release
+
+FROM alpine:3.16.0 as app
+WORKDIR /root/
+# erlexec requires SHELL to be set
+ENV SHELL /bin/sh
+ENV MAGIC_COOKIE=$magic_cookie
+
+RUN apk update && apk add -f openssl libgcc libstdc++ ncurses-libs
```

Comment:
could you add a comment for what each dep is needed for?

Reply:
honestly, they are just there because the Beam needs it... can leave a comment saying that 😅